apps/lib/p5play/p5.play.js

```diff
@@ -448,6 +448,15 @@ p5.prototype._isMouseButtonInState = function(buttonCode, state) {
   if(buttonCode === undefined)
     buttonCode = this.LEFT;
 
+  // Add other acceptable strings
```

Comment:
Right now only constants are accepted for these blocks so I had to add support for strings (which I ended up mapping back to constants). Not sure if this is the right place for these though since I have to modify p5.play, but I couldn't find another place to do it.